src/io/flutter/sdk/FlutterSettingsConfigurable.java

```diff
@@ -230,17 +230,19 @@ public boolean isModified() {
 
   @Override
   public void apply() throws ConfigurationException {
-    final String errorMessage = FlutterSdkUtil.getErrorMessageIfWrongSdkRootPath(getSdkPathText());
-    if (errorMessage != null) {
-      throw new ConfigurationException(errorMessage);
-    }
+    if (!workspaceCache.isBazel()) {
```

Comment:
Is this change part of another modifying the settings page?

Reply:
This was a bug fix for my previous cl. Added a comment clarifying what it is doing.